named_entity_recognition/documentation/step-4-custom-prediction-routine.md

```diff
@@ -0,0 +1,41 @@
+# Custom prediction routine
+
+Allow us to determine which code runs when sending a prediction request.
+Without custom prediction routine the machine learning framework handles the prediction operation.
+With custom prediction routine you can define custom code which runs for each prediction request.
+
+## Why custom prediction routine
+Our model requires numeric inputs, remember the preprocessing steps.
+This is very unhandy if you want to use your model with raw text.
+To support preprocessing also on prediction time you have to define a custom prediction route.
+
+> Without custom prediction routine you would need to implement this preprocessing in an additional wrapper, for example, App Engine or Cloud Function. Which adds complexity and latency.
```

Comment:
How about: Without a custom prediction routine, you would need to create a wrapper, e.g. with App Engine or Cloud Functions, which would add complexity and latency.

Reply:
✓ done